doc/files.md

```diff
@@ -875,18 +875,26 @@ BoxFile file = new BoxFile(api, "id");
 List<String> valueList = new ArrayList<String>();
 valueList.add("bar");
 valueList.add("qux");
-file.updateMetadata(new Metadata().add("/foo", valueList));
+file.updateMetadata(new Metadata("templateScope", "templateKey").add("/foo", valueList));
 ```
 
 If you wanted to replace all selected fields for a specified key you can use the
-[`replace(String key, List<String> values)`][replace-metadata].
+[`replace(String path, List<String> values)`][replace-metadata].
 
 ```java
 BoxFile file = new BoxFile(api, "id");
 List<String> valueList = new ArrayList<String>();
 valueList.add("bar");
 valueList.add("qux");
-file.updateMetadata(new Metadata().replace("/foo", valueList));
+file.updateMetadata(new Metadata("templateScope", "templateKey").replace("/foo", valueList));
+```
+
+If you wanted to remove a metadata value for a specified key you can use the
+[`remove(String path)`][replace-metadata].
```

Comment:
is there no `[remove-metadata]` link equivalent?

Reply:
Nice catch. Just updated. Thanks!